exchange/exchange.go

```diff
@@ -1300,7 +1314,18 @@ func buildStoredAuctionResponse(storedAuctionResponses map[string]json.RawMessag
 			//set imp id from request
 			for i := range seat.Bid {
 				seat.Bid[i].ImpID = impId
-				bidsToAdd = append(bidsToAdd, &entities.PbsOrtbBid{Bid: &seat.Bid[i]})
+				mType := seat.Bid[i].MType
+				var bidType openrtb_ext.BidType
+				if mType > 0 {
+					bidType = openrtb_ext.BidTypes()[mType-1]
```

Comment:
Please validate the mtype is valid (if greater than 0). If it's not valid, consider responding with the same `BadServerResponse` message.

Reply:
Added